src/main/java/apoc/metrics/Metrics.java

```diff
@@ -0,0 +1,111 @@
+package apoc.metrics;
+
+import apoc.ApocConfiguration;
+import apoc.load.LoadCsv;
+import apoc.util.FileUtils;
+import org.neo4j.procedure.Description;
+import org.neo4j.procedure.Name;
+import org.neo4j.procedure.Procedure;
+import java.io.File;
+import java.util.Arrays;
+import java.util.Map;
+import java.util.stream.Stream;
+
+public class Metrics {
+    /**
+     * DAO for a single line in a metrics/*.csv file.
+     * Value is abstracted to double; in reality some are int, some are float, some are double.
+     */
+    public class Neo4jMetricEntry {
+        public long t;
```

Comment:
better name: `timestamp`

Reply:
should we encode the resolution? utcEpocTimeMillis ?

Reply:
I agree that timestamp is a better name but I pick t quite on purpose.  This naming is chosen to match the metrics documentation in the system manual.

The column they expose is t, so I expose t.  I would prefer it be named timestamp, but for avoiding confusion I was thinking it's best to follow the naming convention of what the product docs already say.

Reply:
I think as consumers of the procedure might not know or care where the data comes from and I don't think that the single character naming by the kernel team is such a good idea. I'd rather on the side of clear column names.